apps/04TowersOfHanoi.js

```diff
@@ -16,24 +16,41 @@ function printStacks() {
     console.log("c: " + stacks.c);
 }
 
```

Comment:
So IDK what happened but my original code before commit-ing doesn't look like this... I erased all of the `//Your code here` lines so my code was flush and definitely didn't repeat any unneccesary lines! lol